lib/tasks/webpacker/check_yarn.rake

```diff
@@ -1,15 +0,0 @@
-namespace :webpacker do
-  desc "Verifies if yarn is installed"
-  task :check_yarn do
-    required_yarn_version = "0.20.1"
```

Comment:
Not sure if we are not using it exclusively. I noticed these checks are taking a second or two and slowing down the installer overall. We already documented the version requirements in README anyway and perhaps re-iterate that again inside installation section too?